docs/source/user-guide/concepts/pkg-search.rst

```diff
@@ -40,6 +40,11 @@ example image. The search below will return the same list of packages as the sta
 
    $ conda search "numpy[channel=conda-forge, subdir=linux-64, version=1.17.*, build=py38*]"
 
+This notation supports the following key-value pairs:
+
+.. program-output:: cd .. && python -c 'import os; import sys; src_dir = here = os.path.abspath(os.path.dirname("../setup.py")); sys.path.insert(0, src_dir); import conda.models.match_spec as M; print("-", "\n- ".join([f"{field:20} # validated via {M._implementors.get(field, M.ExactStrMatch).__name__}" for field in sorted(M.MatchSpec.FIELD_NAMES)]))'
```

Comment:
Okay, `setup.py` doesn't exist anymore, should we use `pyproject.toml` to locate the conda folder instead? Why is this needed in the first place?

Reply:
I just copied things from the configuration.rst page. I don't even think `setup.py` is being used at all there; maybe it was just a documentation thing (so we know it's the folder containing setup.py?). `src_dir = os.path.abspath("..")` should be enough.

And I guess the whole thing is to have development `conda` in PYTHONPATH if it's not installed, but maybe this is not needed anymore in the current workflow?

Reply:
I think it's not needed anymore since we've modernized the documentation building last year, but I could be mistaken. Does it still work if you remove it?

Reply:
It works as long as `conda` is installed in the docs environment, in editable mode. Otherwise you get this error, silently:

<img width="860" alt="image" src="https://github.com/conda/conda/assets/2559438/c46ab988-ec7e-4de8-bbcb-29e0889b029b">

Reply:
I guess ReadTheDocs has been configured to install the project in the build env, so I'll remove it.

Reply:
FWIW there was a regression about this after all: #12768